extensions/swagger-ui/deployment/src/main/java/io/quarkus/swaggerui/deployment/SwaggerUiProcessor.java

```diff
@@ -93,9 +93,8 @@ public void getSwaggerUiFinalDestination(
                         "quarkus.swagger-ui.path was set to \"/\", this is not allowed as it blocks the application from serving anything else.");
             }
 
-            String openApiPath = httpRootPathBuildItem.resolvePath(nonApplicationRootPathBuildItem.resolvePath(openapi.path));
```

Comment:
Can `httpRootPathBuildItem` be removed from the method parameters as well?

Reply:
yes. thought I had,